e2e/test/scenarios/embedding/embedding-smoketests.cy.spec.js

```diff
@@ -120,41 +117,14 @@ describe("scenarios > embedding > smoke tests", { tags: "@OSS" }, () => {
 
       cy.log("The second section: 'Interactive embedding'");
       cy.findByTestId("-interactive-embedding-setting").within(() => {
-        const fullAppEmbeddingPath =
-          "/admin/settings/embedding-in-other-applications/full-app";
-
-        cy.findAllByRole("link")
-          .should("have.attr", "href")
-          .and("eq", fullAppEmbeddingPath);
-
-        cy.findByText(/Paid/i);
         cy.findByText("Interactive embedding");
-        cy.findByText(
-          "With this Pro/Enterprise feature, you can let your customers query, visualize, and drill-down on their data with the full functionality of Metabase in your app or website, complete with your branding. Set permissions with SSO, down to the row- or column-level, so people only see what they need to.",
-        );
-        cy.findByText("More details").click();
```

Comment:
This test was also accessing the interactive embedding page that got removed for OSS.
I just left a test for the button, that should link to the metabase website